spec/features/inviting_new_submissions_in_place_of_expired_ones_feature_spec.rb

```diff
@@ -0,0 +1,40 @@
+# Testing rake tasks method found at:
+# http://carlosplusplus.github.io/blog/2014/02/01/testing-rake-tasks-with-rspec/
+# https://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+
+require 'rails_helper'
+require 'rake'
+
+describe 'inviting new submissions in place of expired ones' do
```

Comment:
Another argument against keeping those tests is that we're still not 100% covered because after changing name o the rake task, you can forget to update schedule.rb which is not tested. Unless we test schedule.rb... :smiling_imp: